internal/host/transition.go

```diff
@@ -27,7 +26,6 @@ import (
 type transitionHandler struct {
 	db            *gorm.DB
 	log           logrus.FieldLogger
-	stream        stream.EventStreamWriter
```

Comment:
Was this just not being used?

Reply:
:facepalm: I wonder why the linter didn't pick it up as not used?